### PR TITLE
Update pathology feedback for BS_H1XPVS9A

### PR DESCRIPTION
**Pull request by jharenza**
#### What scientific question is your analysis addressing?
BS_H1XPVS9A was still coded as LGG instead of PXA in the integrated diagnosis, so an update was made here.

#### What was your approach?
I added an additional detail from Mariarita in that she classified this as a malignant PXA after review. Given that the pathology free text diagnosis contains a grade III lesion, it is possible this could still be a high-grade lesion, but cluster with LGG given the BRAF mutational status. This has been noted in the publication [here](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5610026/). At the time of this review, we did not have the pathology report, but just had slides. I will follow up again on this to determine whether it should really be in HGG or LGG. For now, I kept it in LGG since the remaining PXAs are all in LGG (although, they can span grades - we just don't have that level of granularity here).

#### What GitHub issue does your pull request address?
#1093

#### Which areas should receive a particularly close look?
There are additional changes in this PR, presumably due to upstream subtyping tickets (TP53 CNV changes for HGAT, and pathology free text), so I think these are all expected.

#### Is there anything that you want to discuss further?
No

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
Yes

#### What types of results are included (e.g., table, figure)?
Updated table of compiled molecular subtypes

#### What is your summary of the results?
BS_H1XPVS9A now has an integrated diagnosis of `Pleomorphic xanthoastrocytom, BRAF V600E` but remains with LGG histology

#### Reproducibility Checklist

- [ ] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [ ] This analysis has been added to continuous integration.

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

**Comment by jaclyn-taroni** [comment]
@jharenza @kgaonkar6 can someone on your side resolve these conflicts?  I just merged #1097 and I'm not confident about doing so myself.

File: analyses/molecular-subtyping-pathology/04-incorporate-pathology-feedback.Rmd [hunk 1]
```diff
@@ -203,7 +203,7 @@ There are no _BRAF_ fusions in `BS_H1XPVS9A`, so we are able to classify this sa
 compiled_df <- compiled_df %>%
   mutate(
     integrated_diagnosis = case_when(
-      sample_id == "7316-1106" ~ "Low-grade glioma/astrocytoma, BRAF V600E",
+      sample_id == "7316-1106" ~ "Pleomorphic xanthoastrocytom, BRAF V600E",
```
Comment (jaclyn-taroni) [inline analyses/molecular-subtyping-pathology/04-incorporate-pathology-feedback.Rmd:206]:
```suggestion
      sample_id == "7316-1106" ~ "Pleomorphic xanthoastrocytoma, BRAF V600E",
```

Comment (jharenza) [reply analyses/molecular-subtyping-pathology/04-incorporate-pathology-feedback.Rmd:206]: Gahhhhhh!

**Comment by jaclyn-taroni** [comment]
@jharenza I think everything looks in order, but things need to get rerun and committed now that that typo is fixed.

**Review by jaclyn-taroni — APPROVED** [review]

**Pull request merged**